ios/RNManualActivationRecognizer.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIGestureRecognizerSubclass.h>
+
+@class RNGestureHandler;
+
+@interface RNManualActivationRecognizer : UIGestureRecognizer <UIGestureRecognizerDelegate>
```

Comment:
I wonder if having separate gesture recognizer is the best way of handling this. Have you tried looking into using this delegate method: https://developer.apple.com/documentation/uikit/uigesturerecognizerdelegate/1624213-gesturerecognizershouldbegin?language=objc ?

Reply:
I tried to do it this way but `gestureRecognizerShouldBegin` is called at the very beginning of the gesture lifecycle and if you return `NO` the gesture immediately fails to recognize.